tools/pot/configs/README.md

```diff
@@ -34,21 +29,19 @@ This section contains only three parameters:
         "config": "./configs/examples/accuracy_checker/mobilenet_v2.yaml"
     }
 ```
-The main parameter is `"type"` which can take two possible options: `"accuracy_checher"` (default) and `"simplified"`,
-which specify the engine that is used for model inference and validation (if supported):
-- **Simplified mode** engine. This engine can be used only with `DefaultQuantization` algorithm to get fully quantized model 
-using a subset of images. It does not use the Accuracy Checker tool and annotation. To measure accuracy, you should implement 
-your own validation pipeline with OpenVINO API.  
-  - To run the simplified mode, define engine section similar to the example `mobilenetV2_tf_int8_simple_mode.json` file from the `<POT_DIR>/configs/examples/quantization/classification/` directory.
-- **Accuracy Checker** engine. It relies on the [Deep Learning Accuracy Validation Framework](@ref omz_tools_accuracy_checker) (Accuracy Checker) when inferencing DL models and working with datasets.
+The main parameter is `"type"` which can take two possible options: `"accuracy_checher"` (default) or `"simplified"`. It specifies the engine that is used for model inference and validation (if supported):
+- **Simplified mode** engines. These engines can be used only with `DefaultQuantization` algorithm to get fully quantized model. They do not use the Accuracy Checker tool and annotation. In the case of this mode the following parameters are applicable:
+  - `"data_source"` Specifies path to directory​ where to calibration data is stored.
+  - `"layout"` - (Optional) Layout of input data. Supported values: [`"NCHW"`, `"NHWC"`, `"CHW"`, `"CWH"`]​.
+- **Accuracy Checker** engine. It relies on the [Deep Learning Accuracy Validation Framework](@ref omz_tools_accuracy_checker_README) (Accuracy Checker) when inferencing DL models and working with datasets.
 The benefit of this mode is you can compute accuracy in case you have annotations. It is possible to use accuracy aware
```

Comment:
It is possible to use accuracy aware ---> When this mode is selected, you can use the accuracy aware algorithms family.